pkg/client/userd/trafficmgr/agents.go

```diff
@@ -51,6 +58,30 @@ func (s *session) getCurrentAgentsInNamespace(ns string) map[string]*manager.Age
 	return agents
 }
 
+func (s *session) getCurrentSidecarsInNamespace(ctx context.Context, ns string) map[string]*agentconfig.Sidecar {
+	sidecars := make(map[string]*agentconfig.Sidecar)
+
+	// Load configmap entry from the telepresence-agents configmap
+	cm, err := k8sapi.GetK8sInterface(ctx).CoreV1().ConfigMaps(ns).Get(ctx, agentconfig.ConfigMap, meta.GetOptions{})
+	if err != nil {
+		if !k8sErrors.IsNotFound(err) {
+			dlog.Error(ctx, errcat.User.New(err))
+		}
+		return sidecars
+	}
+
+	for workload, sidecar := range cm.Data {
```

Comment:
Can data be nil ?

Reply:
yes, but it does panic the cli. I added a debug log